meson.build

```diff
@@ -130,6 +130,7 @@ conf.set('HAVE_MEMORY_H', cc.has_header('memory.h'))
 
 # Functions
 conf.set('HAVE_ALIGNED_ALLOC', cc.has_function('aligned_alloc', prefix: '#include <stdlib.h>'))
+conf.set('HAVE__ALIGNED_MALLOC', cc.has_function('_aligned_malloc', prefix: '#include <stdlib.h>'))
```

Comment:
I thought on MinGW it was only defined in `stdlib.h` so I went with that. It turns out that it is defined in both `malloc.h` and `stdlib.h` for MinGW. Not sure about MSVC, but the safe bet is that it's in `malloc.h`. I guess we should change this. I'll submit a new PR since there's one more change that's needed with this.